tests/test_train.py

```diff
@@ -430,30 +431,44 @@ def test_model_finetuning_nlu(
     monkeypatch: MonkeyPatch,
     default_domain_path: Text,
     default_nlu_data: Text,
-    default_stack_config: Text,
-    trained_rasa_model: Text,
+    trained_moodbot_path: Text,
```

Comment:
I believe we should also update the test for `test_model_finetuning`

Reply:
I thought it would make sense to do it once both nlu and core were done.

Reply:
Ok, then let's add a `TODO` so we don't forget